tests/ui/doc/doc.stderr

```diff
@@ -1,160 +1,160 @@
-error: you should put `foo_bar` between ticks in the documentation
+error: item in documentation missing backticks
   --> $DIR/doc.rs:8:9
    |
 LL | /// The foo_bar function does _nothing_. See also foo::bar. (note the dot there)
-   |         ^^^^^^^
+   |         ^^^^^^^ help: try: ``foo_bar``
```

Comment:
Is there a reason why the suggestion uses two backticks? Markdown usually handles one and two backticks the same:

```md
Hello `how` are ``you`` doing?
```
-> Hello `how` are ``you`` doing?

Reply:
All suggestions are put in backticks, but they aren't actually included in the fix. I don't know how to make it a single pair and still include them.

Reply:
Makes sense! Thank you :+1: